pdns/dnsdistdist/docs/guides/tls-sessions-management.rst

```diff
@@ -0,0 +1,56 @@
+TLS Sessions Management
+=======================
+
+TLS sessions
+------------
+
+One of the most costly TLS operation if the negotiation of a new session, since both the client and the server need to generate and agree on cryptographic materials. In order to reduce that cost, TLS implements what is called session resumption, where a client opening a new connection to a server can reuse the cryptographic materials negotiated for a previous TLS session.
+
+The necessary information to resume a session can either be kept on the server's side (sessions) or on the client's one (tickets). Initially only the server-side approach existed, with two drawbacks:
+- the server needs to keep that information at hand, for a client that might never come back;
+- sharing that information between several servers is not easy, especially in setups involving anycast or any kind of cluster without strong session affinity.
+
+Nowadays pretty much all clients support the second option, TLS tickets, where the need information is signed and encrypted by the server before being sent to the client, which is responsible for storing it and sending it back when it wants to establish a new session. That reduces the burden of the server while providing the same benefits.
+
+The server uses Session Ticket Encryption Key (STEK) to sign and encrypt the information sent to the client, making it possible to ensure that it is genuine and has not been tampered when the client provides it later. That STEK can be shared by all dnsdist instances in the same cluster, making it possible for any server to resume a session initially generated by a different server.
+
+Knowing the STEK is all the information needed to be able to decrypt a live TLS session, but also a recorded one, so it is very important to keep that key well-protected. It should never be exchanged in clear-text, and ideally should not be written to persistent storage but be kept in a tmpfs with no swap configured. It should also be regularly rotated to preserve TLS' forward secrecy properties.
+
+Keys management in dnsdist
+--------------------------
+
```

Comment:
dnsdist only support TLS t tickets, right? Please mention that if so.

Reply:
It supports both server-side sessions and client-side tickets, although my advice is to only use the latter.